index.ts

```diff
@@ -0,0 +1,11 @@
+export * as casingData from './casings.json';
+export * as cementData from './cement.json';
+export * as completion from './completion.json';
+export * as coordinates from './coordinates.json';
+export * as holeSizeData from './holesize.json';
+export * as picks from './picks.json';
+export * as poslog from './poslog.json';
+export * as seismicColorMap from './seismic-colormap.json';
+export * as seismic from './seismic.json';
+export * as stratColumns from './strat-columns.json';
+export * as surfaces from './surfaces.json';
```

Comment:
I assume we should standardize the naming strategy here, but I'd rather do it afterwards, after we have checked that everything works properly on radix as well